pkgs/os-specific/linux/kernel/patches.nix

```diff
@@ -57,4 +57,31 @@ rec {
       sha256 = "1l8xq02rd7vakxg52xm9g4zng0ald866rpgm8kjlh88mwwyjkrwv";
     };
   };
+
+  ck_4_14 = rec {
+    name = "patch-4.14-ck1.xz";
+    patch = fetchpatch {
+      name   = name + ".patch";
+      url    = http://ck.kolivas.org/patches/4.0/4.14/4.14-ck1/patch-4.14-ck1.xz;
+      sha256 = "76863d16a08e1b3c726b6c965f1bb7c672bd7317776810121062b73f9ea26780";
+    };
+  };
+
+  ck_4_19 = rec {
+    name = "patch-4.19-ck1.xz";
+    patch = fetchpatch {
+      name   = name + ".patch";
+      url    = http://ck.kolivas.org/patches/4.0/4.19/4.19-ck1/patch-4.19-ck1.xz;
+      sha256 = "77863d16a08e1b3c726b6c965f1bb7c672bd7317776810121062b73f9ea26780";
+    };
+  };
+
+  ck_5_0 = rec {
+    name = "patch-5.0-ck1.xz";
+    patch = fetchpatch {
```

Comment:
```
error: Normalized patch '/build/76863d16a08e1b3c726b6c965f1bb7c672bd7317776810121062b13f9ea26780' is empty (while the fetched file was not)!
Did you maybe fetch a HTML representation of a patch instead of a raw patch?
Fetched file was:
...<BINARY>
```

Can't apply because  it's still an archive.

Reply:
I have the time and resources to test-build this now... so wait for it, I'm tryin' it out.

Reply:
So yes, the problem is indeed that I fetch .xz and not .patch ... I am working on a solution right now.